cluster/images/pipeline5.cmds

```diff
@@ -34,6 +34,7 @@ sudo chmod a+x /opt/tools/bwa/0.7.17/bwa
 sudo chmod a+x /opt/tools/sambamba/0.6.5/sambamba
 sudo chmod a+x /opt/tools/sambamba/0.6.8/sambamba
 sudo chmod a+x /opt/tools/chord/60.02_1.03/extractSigPredictHRD.R
+sudo chmod a+x /opt/tools/chord/2.00_1.14/extractSigPredictHRD.R
```

Comment:
With the wonky versioning might it make sense to remove the `60.02_1.03` version now so it doesn't get kept instead of the new one in a later cleanup?

Reply:
should be fine, i don't think we'd ever remove a version solely based on increments